mod/settings.php

```diff
@@ -1106,40 +1106,40 @@ function settings_content(App $a)
 		$profile_in_dir = '<input type="hidden" name="profile_in_directory" value="1" />';
 	} else {
 		$profile_in_dir = replace_macros($opt_tpl, [
-			'$field' => ['profile_in_directory', L10n::t('Publish your default profile in your local site directory?'), $profile['publish'], L10n::t("Your profile may be visible in public."), [L10n::t('No'), L10n::t('Yes')]]
+			'$field' => ['profile_in_directory', L10n::t('Publish your default profile in your local site directory?'), $profile['publish'], L10n::t('If you enable this option your profile will be published in the global friendica directories (e.g. <a href="%s">%s</a>). Your profile may be visible in public.', Config::get('system', 'directory'), Config::get('system', 'directory')), [L10n::t('No'), L10n::t('Yes')]]
 		]);
 	}
 
 	if (strlen(Config::get('system', 'directory'))) {
 		$profile_in_net_dir = replace_macros($opt_tpl, [
-			'$field' => ['profile_in_netdirectory', L10n::t('Publish your default profile in the global social directory?'), $profile['net-publish'], '', [L10n::t('No'), L10n::t('Yes')]]
+			'$field' => ['profile_in_netdirectory', L10n::t('Publish your default profile in the global social directory?'), $profile['net-publish'], L10n::t('If you enable this option your profile will be listed in this nodes <a href="%s">local directory</a>.', System::baseUrl().'/directory'), [L10n::t('No'), L10n::t('Yes')]]
```

Comment:
I believe you swapped the help texts between `profile_in_directory` and `profile_in_netdirectory`.

`'Your profile will be published in this node\'s <a href="%s">local directory</a>. Your profile details will be publicly visible.', System::baseUrl().'/directory'`

Reply:
> I believe you swapped the help texts between profile_in_directory and profile_in_netdirectory.

Why? Then they are in the wrong order for a long time, as I have only added the missing help text.

Reply:
The details still may be visible for the public, not will be, depending on how the admin settings of the node are. Public meaning unknown visitors from the internet.